gecko-media/CMakeLists.txt

```diff
@@ -923,7 +983,7 @@ if(LINUX)
     gecko/src/media/libcubeb/src/cubeb_pulse.c
   )
 endif()
-if(MACOS)
+if(MACOS AND NOT ANDROID)
```

Comment:
I'd have thought that MACOS and ANDROID would be mutually exclusive? If not, it's probably best to make them so.

Reply:
They're not because MACOS, LINUX and WINDOWS currently match the host system OS, if I understand correctly. And ANDROID can match the target. Host vs target when cross-compiling :)

Reply:
Maybe we should rename ANDROID to ANDROID_TARGET or some more explicit name? I can do it as a follow-up of this PR.